tests/PKSim.Tests/Core/GlobalPKAnalysisTaskSpecs.cs

```diff
@@ -29,7 +30,7 @@ public abstract class concern_for_GlobalPKAnalysisTask : ContextSpecification<IG
       private IVSSCalculator _vssCalculator;
       protected DataColumn _peripheralVenousBloodPlasma;
       protected DataColumn _venousBloodPlasma;
-      protected IndividualSimulation _simulation;
```

Comment:
I refactored this test a bit because the base abstract class was setting up everything for an individual simulation, then the population_based was subclassing that and adding some population simulations.  When I wrote my tests, I couldn't figure out why they were failing. The assertions were using the wrong simulation!